specs/trip_spec.rb

```diff
@@ -38,5 +41,16 @@
         }.must_raise ArgumentError
       end
     end
+
+    it "returns trip duration" do
+      @trip.trip_duration.must_be_kind_of Integer
+      @trip.trip_duration.must_equal 25
+    end
+
+    it "raises an ArgumentError" do
+      proc {
+        @trip_data[:end_time] = '2015-05-20T12:00:00+00:00'
+        RideShare::Trip.new(@trip_data).trip_duration}.must_raise ArgumentError
```

Comment:
This test just says that the method "should raise an ArgumentError," but doesn't explain why, and I can't tell just from this code _why_ this method raises one (especially from a business perspective). Maybe add to the it description "raises an ArgumentError _when end time is ..._" or use a local variable name to describe what is going on (invalid_time?)